README.md

```diff
@@ -399,6 +399,39 @@ app.get('/auth', passport.authenticate('oidc', [options]));
 app.get('/auth/cb', passport.authenticate('oidc', { successRedirect: '/', failureRedirect: '/login' }));
 ```
 
+### Passport with PKCE
+The Client instance must have the property `token_endpoint_auth_method` setted to `none`.
```

Comment:
The client instance does not have to have a specific `token_endpoint_auth_method` to use PKCE. As a matter of fact there are OIDC profiles that require the use of PKCE even for confidential authorization_code clients.